lib/components/Dialog/DialogCard.treat.ts

```diff
@@ -0,0 +1,47 @@
+import { style } from 'sku/treat';
```

Comment:
How would you feel about putting all the styles in `Dialog.treat` rather than two separate files?